src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzerTests.cs

```diff
@@ -128,6 +109,106 @@ class Target
             await VerifyAnalyzerCSAsync(source, tfmAndOption);
         }
 
+        [Fact]
+        public async Task GeneratedFile_RazorWarns()
```

Comment:
Just to confirm - did you verify this test fails before your product side changes? The auto-generated comment is not the first line in the file, so it would be good to double check the test failure prior to your changes.

Reply:
Do not remember if I did, anyway I can check that

Reply:
it fails without changes, and also moving the `auto-generated` comment other rows works